### PR TITLE
Move libraries installation to separate template

### LINKED ISSUE #53: Specifying libraries for a specific env seems broken.
See https://dev.azure.com/OpenAstronomy/azure-pipelines-templates/_build/results?buildId=282&view=results and associated branch

### PR DESCRIPTION
**Pull request by ConorMacBride**
Closes #53

### CODE REVIEW

**Comment by ConorMacBride** [comment]
@Cadair This should be working now. Although, it should be tested elsewhere before merging, if you think this approach is good.

**Comment by Cadair** [comment]
This looks like a really neat solution. I like the splitting up of the monolithic tox template into sub-templates, it's a nice pattern.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -41,6 +41,18 @@ stages:
         - linux: testtoxargs
           toxargs: --notest
 
+    - template: run-tox-env.yml
+      parameters:
+        libraries:
+          apt:
+          - rolldice
```
Comment (Cadair) [inline azure-pipelines.yml:48]: :clap:

**Comment by ConorMacBride** [comment]
@Cadair Merge?

**Review by Cadair — APPROVED** [review]

**Pull request merged**